dev.py

```diff
@@ -431,6 +431,11 @@ class Build(Task):
         ['--with-scipy-openblas'], default=False, is_flag=True,
         help=("If set, use the `scipy-openblas32` wheel installed into the "
               "current environment as the BLAS/LAPACK to build against."))
+    with_accelerate = Option(
+        ['--with-accelerate'], default=False, is_flag=True,
+        help=("If set, use `Accelerate` as the BLAS/LAPACK to build against."
+              " Takes precedence over -with-scipy-openblas (macOS only)")
```

Comment:
Might be a dash missing:

```suggestion
              " Takes precedence over --with-scipy-openblas (macOS only)")
```

That said, it seems to work just fine locally when testing on my ARM mac with various incantations, and seems to be useful, so I'll go ahead and put it in, thanks.